api_backend.yaml

```diff
@@ -437,6 +450,46 @@ paths:
           schema:
             $ref: "#/definitions/ProblemJson"
 definitions:
+  # Custom Definitions
+  GetPaymentInfoProxyRequest:
+    type: object
+    title: GetPaymentInfoProxyRequest
+    description: Request object for getPaymentInfo
+    properties:
+      rptId:
+        $ref: "#/definitions/RptId"
+      test:
```

Comment:
this parameter is already part of the query string, why you need it also in the payload?